lib/commands/projects.js

```diff
@@ -2168,19 +2514,34 @@ projects
 
 projects
     .command(`update-api-status`)
-    .description(`Update the status of a specific API type. Use this endpoint to enable or disable API types such as REST, GraphQL and Realtime.`)
+    .description(`[**DEPRECATED** - This command is deprecated. Please use 'projects update-api-status' instead] Update the status of a specific API type. Use this endpoint to enable or disable API types such as REST, GraphQL and Realtime.`)
```

Comment:
Same deprecated problem as mentioned before, we should check all places where DEPRECATED appears